projects/demo/src/modules/components/input-year/examples/1/index.ts

```diff
@@ -1,16 +1,17 @@
 import {Component} from '@angular/core';
-import {FormControl, ReactiveFormsModule} from '@angular/forms';
+import {FormsModule} from '@angular/forms';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
-import {TuiInputYearModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';
+import {TuiCalendarYear, TuiTextfield} from '@taiga-ui/core';
+import {TuiInputYear} from '@taiga-ui/kit';
 
 @Component({
     standalone: true,
-    imports: [ReactiveFormsModule, TuiInputYearModule, TuiTextfieldControllerModule],
+    imports: [FormsModule, TuiCalendarYear, TuiInputYear, TuiTextfield],
```

Comment:
Should `TuiInputYear` include `TuiCalendarYear`? I believe other pickers do that.

Reply:
fixed